src/main/java/org/gridsuite/modification/server/repositories/NetworkModificationRepository.java

```diff
@@ -246,13 +248,51 @@ public EquipmentCreationEntity createLineEntity(String lineId, String lineName,
     }
 
     public EquipmentCreationEntity createTwoWindingsTransformerEntity(String id, String name, double seriesResistance, double seriesReactance,
```

Comment:
Too many parameters. At least separate phaseTapChanger and ratioTapChanger in separate structures (info, entity class, tables). Other win in this move : name of fields will be shorter

Reply:
As discussed, I'll create a technical story about this